### ISSUE TITLE
[BUG] _HAS_DEFAULT_FACTORY is not defined

### ISSUE THREAD

**Issue by Mallikarjun-49** [Arize-ai/phoenix#1041]
Hi phoenix team,
I am facing issues when I run from my local vscode
 
Error fetching GraphQL query 'EmbeddingUMAPQuery' with variables '{"id":"RW1iZWRkaW5nRGltZW5zaW9uOjA=","timeRange":{"start":"2022-08-03T20:30:00.000Z","end":"2022-08-05T20:30:00.000Z"},"minDist":0,"nNeighbors":30,"nSamples":500,"minClusterSize":10,"clusterMinSamples":1,"clusterSelectionEpsilon":0,"fetchDataQualityMetric":false,"fetchPerformanceMetric":false,"dataQualityMetricColumnName":null,"performanceMetric":"accuracyScore"}': [{"message":"name '_HAS_DEFAULT_FACTORY' is not defined","locations":[{"line":18,"column":7}],"path":["embedding","UMAPPoints"]}]

**Comment by mikeldking** [comment]
@Mallikarjun-49 Thanks so much for the ticket. You are the second person running into this issue - it could be windows related. Can you confirm that this is the case? Thanks again for contacting us!

**Comment by mikeldking** [comment]
duplicate of [1023](https://github.com/Arize-ai/phoenix/issues/1023)

**Comment by mikeldking** [comment]
@Mallikarjun-49 Any chance you could provide us with:

- OS
- output of pip freeze
- your python version

Thanks! We will try to unblock you in the next week or so.
